.github/workflows/deploy.yml

```diff
@@ -0,0 +1,16 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    steps:
+      - uses: rust-lang/simpleinfra/github-actions/static-websites@master
+        with:
+          deploy_dir: /path/to/output
```

Comment:
@pietroalbini I'm not sure what to put here :thinking:

Reply:
Seems we need to use `$RUSTINFRA_DEPLOY_DIR`.

https://github.com/rust-lang/simpleinfra/blob/421f6a4429c8e6a60349ffd5f5192a89892ff08d/travis-configs/static-websites.yml#L28

Reply:
great suggestion, thanks! I've pushed the change